buildSrc/src/main/java/Config.kt

```diff
@@ -3,7 +3,7 @@ object Config {
     val kotlinStdLib = "stdlib-jdk8"
 
     object BuildPlugins {
-        val androidGradle = "com.android.tools.build:gradle:3.5.2"
+        val androidGradle = "com.android.tools.build:gradle:3.6.0-beta03"
```

Comment:
@ninniuz what do you think about that?
I'd like to use `maven-publish` with the native support that AGP 3.6.x provides.

Reply:
It is a great addition to AGP and it makes publishing a lot easier, but I wouldn't use beta releases as they could affect aar generation.

Reply:
@ninniuz based on the changelog, the "only" different thing that affects aar generation would be `useNewApkCreator` which I've disabled, that's why I think it's safe to go, but I do understand your concern.